cmd/kar-controllers/app/options/options.go

```diff
@@ -48,6 +50,8 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to kubeconfig file with authorization and master location information.")
 	fs.BoolVar(&s.Dispatcher,"dispatcher",s.Dispatcher,"set dispather mode(true) or agent mode(false)")
 	fs.StringVar(&s.AgentConfigs, "agentconfigs", s.AgentConfigs, "Paths to agent config file:deploymentName separted by commas(,)")
+	fs.BoolVar(&s.DynamicPriority,"dynamicpriority", s.DynamicPriority,"--dynamicpriority=false/true set controller use dynamic priority")
+	fs.BoolVar(&s.Preemption,"preemption", s.Preemption,"--preemption=false/true set controller to allow preemption")
```

Comment:
I realized that this option comes with configuration requirements.  The MCAD controller does not do the actual preemption of pods.  Thus in order for preemption to be realized in a cluster the scheduler must be configured to preempt.  Suggested additional text: `--preemption=false/true set controller to allow preemption.  Note: when set to true, the Kubernetes Scheduler must be configured to enable preemption.`